test/jdk/ProblemList.txt

```diff
@@ -805,3 +805,182 @@ java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter.java 8254841 macos
+java/lang/Thread/virtual/ThreadLocals.java                      8286642 generic-i586
+java/lang/Thread/virtual/TracePinnedThreads.java                8286642 generic-i586
+java/lang/Thread/virtual/WaitNotify.java                        8286642 generic-i586
+java/lang/Thread/virtual/stress/GetStackTraceALot.java#id0      8286642 generic-i586
+java/lang/Thread/virtual/stress/GetStackTraceALot.java#id1      8286642 generic-i586
+java/lang/Thread/virtual/stress/PinALot.java#id0                8286642 generic-i586
+java/lang/Thread/virtual/stress/PinALot.java#id1                8286642 generic-i586
+java/lang/Thread/virtual/stress/Skynet.java#id0                 8286642 generic-i586
+java/lang/Thread/virtual/stress/Skynet.java#id1                 8286642 generic-i586
+java/lang/Thread/virtual/stress/SleepALot.java#id0              8286642 generic-i586
+java/lang/Thread/virtual/stress/SleepALot.java#id1              8286642 generic-i586
+java/lang/Thread/virtual/stress/YieldALot.java#id0              8286642 generic-i586
+java/lang/Thread/virtual/stress/YieldALot.java#id1              8286642 generic-i586
+java/net/Socket/Timeouts.java                                   8286642 generic-i586
+java/net/vthread/BlockingSocketOps.java                         8286642 generic-i586
+java/net/vthread/InterruptHttp.java                             8286642 generic-i586
+java/nio/channels/FileChannel/LargeMapTest.java                 8286642 generic-i586
+java/nio/channels/FileChannel/MapToMemorySegmentTest.java       8286642 generic-i586
+java/nio/channels/vthread/BlockingChannelOps.java               8286642 generic-i586
+java/util/concurrent/ExecutorService/CloseTest.java             8286642 generic-i586
+java/util/concurrent/ThreadPerTaskExecutor/ThreadPerTaskExecutorTest.java               8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/lang/invoke/DeserializeMethodTest.java     8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/lang/invoke/MHProxiesTest.java             8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/lang/invoke/SerializedLambdaTest.java      8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/FillableStringTest.java               8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/MapTest.java                          8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/SplittableRandomTest.java             8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/stream/CollectAndSummaryStatisticsTest.java   8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/stream/CollectionAndMapModifyStreamTest.java  8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/stream/CollectorExample.java          8286642 generic-i586
+java/util/stream/test/org/openjdk/tests/java/util/stream/CollectorToUnmodListTest.java  8286642 generic-i586
```

Comment:
I'm surprised that the stream tests are included as they don't run with --enable-preview.

Reply:
Those tests *do* run with preview enabled: https://github.com/openjdk/jdk/blob/master/test/jdk/java/util/stream/test/TEST.properties#L10-L11 -- and I know that because I basically copy-pasted all jtreg failures that lead to `Unimplemented()` in x86_32 code.

Reply:
Yes, `TEST.properties` is way too broad. I'd keep the current patch as is, instead of trying to fix that one. I suspect there is some funky TestNG/othervm interaction going...

Reply:
These specific stream tests are odd, in the sense that they live in a test root that is TestNG and they are only "imported" into jtreg. The toplevel TEST.properties file gets only once chance to set the correct parameters, for all tests. I've tried adding more local flags, nearer to the test, w/o success. I'm obviously happy to do it another way.

I have tried adding a jtreg header in that test, but that doesn't fix the problem - because _all_ the tests are passed to testng at once, which compiles them and runs them. So either the flag is present when that happens, or it is ignored.